dev/com.ibm.ws.concurrent.persistent_fat_multiple/fat/src/com/ibm/ws/concurrent/persistent/fat/multiple/DatabaseContainerType.java

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.concurrent.persistent.fat.multiple;
+
+/**
+ * This is a current list of databases that are in the database rotation.
+ *
+ * Each database container type has three values associated with it: <br>
+ * 1. String name - Matches the value of the system property {fat.bucket.db.type} when enlisting database rotation.<br>
+ * 2. String driverName - Matches the common name given to the JDBC driver jar: '{driverName}.jar'.<br>
+ * 3. Boolean haveContainer - Answers whether or not there is a test container for this database.<br>
+ *
+ * TODO: Since this enum is general it should be moved to fattest simplicity in the future
+ */
+public enum DatabaseContainerType {
+    DERBY("Derby", "derby.jar", true),
+    DB2("DB2", "db2jcc.jar", true),
+    ORACLE("Oracle", "ojdbc8_g.jar", true),
+    POSTGRE("Postgre", "postgresql.jar", true),
+    SQLSERVER("SQLServer", "mssql-jdbc.jar", false), //FIXME change to true once we solve XA connection issues
+    INFORMIX("Informix", "ifxjdbc.jar", false),
+    SYBASE("Sybase", "jconn4.jar", false);
+
+    private String name;
+    private String driverName;
+    private boolean hasTestContainer;
+
+    DatabaseContainerType(final String name, final String driverName, final boolean inService) {
+        this.name = name;
+        this.driverName = driverName;
+        this.hasTestContainer = inService;
```

Comment:
this boolean and corresponding methods can be removed, since we can get rid of informix and sybase and just comment out the sqlserver code for now